README.md

```diff
@@ -26,7 +27,7 @@ https://user-images.githubusercontent.com/60359573/230078733-5faffa19-e6ce-4c55-
 </div>
 
 ## Pretrained models
-
+<div align="center">
 <table>
```

Comment:
After doing a bit of research, it seems styling the table to auto-center is the proper way to achieve this:

```
<table style="margin: auto">
```